### PR TITLE
Doc: HPC3 at UCI

### LINKED ISSUE #3938: WarpX on new HPC cluster at UCI
Hello, I'm trying to install and run Warpx on my institution's cluster. I have some experience installing softwares but there are some dependancies that I'm not sure what they do or how to get them. Thank you for the help !

### PR DESCRIPTION
**Pull request by ax3l**
Document how to install and run GPU-accelerated WarpX on HPC3 (UCI).

Developed with @floresv299 :tada: 

Close #3938

### CODE REVIEW

File: Docs/source/install/hpc/hpc3.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+As a last step, clean the build directory ``rm -rf $HOME/src/warpx/build`` and rebuild WarpX.
+
+
+.. _running-cpp-hpc3:
+
+Running
+-------
+
+The batch script below can be used to run a WarpX simulation on multiple nodes (change ``-N`` accordingly) on the supercomputer HPC3 at UCI.
+This partition as up to `ABC nodes <https://rcic.uci.edu/hpc3/slurm.html#memmap>`__ with four V100 GPUs per node.
```
Comment (ax3l) [inline Docs/source/install/hpc/hpc3.rst]: @floresv299 let's complete this based on https://github.com/ECP-WarpX/WarpX/issues/3938#issuecomment-1597467944 and https://github.com/ECP-WarpX/WarpX/issues/3938#issuecomment-1597482555
```suggestion
This partition as up to `N nodes <https://rcic.uci.edu/hpc3/slurm.html#memmap>`__ with four V100 GPUs (.. GB each) per node.
```

File: Tools/machines/hpc3-uci/hpc3_gpu.sbatch [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# License: BSD-3-Clause-LBNL
+
+#SBATCH --time=00:30:00
+#SBATCH --nodes=1
+#SBATCH -J WarpX
+#S BATCH -A <proj>
+# V100 GPU options: gpu, free-gpu, debug-gpu
+#SBATCH -p free-gpu
+#SBATCH --cpus-per-task=10
+#SBATCH --gres=gpu:V100:4
```
Comment (ax3l) [inline Tools/machines/hpc3-uci/hpc3_gpu.sbatch]: @floresv299 we need to test if this properly uses all four GPUs. I have not yet found so-called Slurm "pinning" options, where we assign the MPI rank to the CPUs that are closest to each of the four GPUs. Something we need to:
- test by running 1, 2, 4 GPUs (1 node) and 8 GPUs (2 nodes) and
- need to write an email to your HPC3 support about :)

File: Docs/source/install/hpc/hpc3.rst (hunk 1 above)
Comment (ax3l) [reply Docs/source/install/hpc/hpc3.rst]: Ok, I count 32 notes in https://github.com/ECP-WarpX/WarpX/issues/3938#issuecomment-1597554390 with 16 GB each https://github.com/ECP-WarpX/WarpX/issues/3938#issuecomment-1597564390 :)

File: Tools/machines/hpc3-uci/hpc3_gpu.sbatch (hunk 2 above)
Comment (ax3l) [reply Tools/machines/hpc3-uci/hpc3_gpu.sbatch]: I sent an email to UCI HPC support to square out the last question on system-specific control of GPU affinity.

@floresv299 a small wish to easier identify you on GitHub: Do you mind setting your name and affiliation in your profile https://github.com/settings/profile ? :)

**Comment by ax3l** [comment]
I'll merge this first version in, so it can be rendered to https://warpx.readthedocs.io/en/latest/install/hpc/hpc3.html

We will do follow-up PRs with updates to the job script :)

File: Tools/machines/hpc3-uci/hpc3_gpu.sbatch (hunk 2 above)
Comment (floresv299) [reply Tools/machines/hpc3-uci/hpc3_gpu.sbatch]: Sure thing Axel, just did. Thank you once again for all your help, you have been amazing !

Comment (ax3l) [reply Tools/machines/hpc3-uci/hpc3_gpu.sbatch]: Thanks, you are most welcome!

**Pull request merged**